cli/lsp/documents.rs

```diff
@@ -929,7 +924,16 @@ impl Documents {
 
   /// Return `true` if the specifier can be resolved to a document.
   pub fn contains_specifier(&self, specifier: &ModuleSpecifier) -> bool {
-    self.get(specifier).is_some()
+    let specifier = self.specifier_resolver.resolve(specifier);
+    if let Some(specifier) = specifier {
+      if self.open_docs.contains_key(&specifier) {
+        return true;
+      }
+      if let Some(path) = get_document_path(&self.cache, &specifier) {
+        return path.is_file();
+      }
+    }
+    false
```

Comment:
I moved the code from `exists` here, but I still think we should just do `self.get(specifier).is_some()` because it's simpler and uses the same code path as `.get` so it future proofs this method. In tsc when it calls `exists`, if true then it will cause `.get` to be called so there's not really any reason to not just load it the first time in my opinion.